src/DataViews/ModulesDataViewTest.cpp

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/strings/str_format.h>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <filesystem>
+#include <string>
+
+#include "ClientData/ProcessData.h"
+#include "DataViews/DataView.h"
+#include "DataViews/ModulesDataView.h"
+#include "DisplayFormats/DisplayFormats.h"
+#include "MockAppInterface.h"
+#include "OrbitBase/File.h"
+#include "OrbitBase/ReadFileToString.h"
+#include "OrbitBase/TemporaryFile.h"
+#include "OrbitBase/TestUtils.h"
+#include "module.pb.h"
+
+using orbit_client_data::ModuleData;
+using orbit_client_data::ModuleInMemory;
+using orbit_grpc_protos::ModuleInfo;
+
+namespace {
+
+constexpr size_t kNumModules = 3;
+constexpr std::array<uint64_t, kNumModules> kStartAddresses{0x1000, 0x2000, 0x3000};
+constexpr std::array<uint64_t, kNumModules> kEndAddresses{0x1100, 0x2100, 0x3100};
+constexpr std::array<uint64_t, kNumModules> kSizes{100, 200, 300};
```

Comment:
Maybe shuffle the size around such that the order is not the same as the address ranges. Otherwise the sorting test won't find out if the data view sorts by the wrong column because all sortings are the same.

Reply:
Ah, make sense! Changed now.